### DISCUSSION TITLE
No content is being saved in new install

### DISCUSSION THREAD

**Discussion by avanwieringen-sensus** [directus/directus#3659] [Archive]
I used `npx create-directus-project` to create a fresh Directus install to try out, on a fresh DB.
I created a simple collection, `news` with only ID, Title (string) en Content (text).
I added one item and pressed save, and the item does not appear in the collection. No filters are active.

What am I doing wrong? This is the most basic use-case right?

**Comment by rijkvanzanten** [comment]
Feel free to [open an issue](https://github.com/directus/directus/issues/new) explaining your setup and config in as much detail as possible if you think you ran into a problem. From what you describe here, it sounds normal, but there's a lot of unknowns here 🙂

**Reply by avanwieringen-sensus** [reply]
Could you elaborate why it is normal that no item appears?

It is the most clean install and I add an item to the collection. It does not appear.
I'll create a working example.

**Reply by rijkvanzanten** [reply]
I'm saying that it does _not_ sound normal, but that I don't have enough information about your setup to tell you _why_ it's happening 🙂  Could be a permissions thing, a webserver thing, a browser quirk, a cache thing, an API thing, a configuration thing etc etc
